cmd/svcat/class/get_cmd.go

```diff
@@ -65,6 +75,8 @@ func (c *getCmd) Validate(args []string) error {
 	if len(args) > 0 {
 		if c.lookupByUUID {
 			c.uuid = args[0]
+		} else if c.broker != "" {
+			c.broker = args[0]
```

Comment:
This is working against using StringVarP when parsing the --broker flag. Either get the name of the broker via the StingVarP or get it here, but don't do both. Also, this seems to break down if the --uuid flag and the --broker flag are both used. If it is not acceptable to use both together, then the user should be informed that they supplied incompatible flags when they try to.

Reply:
Anytime a value is configured with a flag, you should use the bound variable, and not retrieve a positional argument. This isn't doing what you are hoping. 😀